docs/api/rest/config.md

```diff
@@ -209,117 +149,143 @@ curl --location --request GET 'http://localhost:5000/api/config/sourceaetitle'
 
 ### Responses
 
-Response Content Type: JSON - Array of [DestinationApplicationEntity](xref:Nvidia.Clara.DicomAdapter.API.DestinationApplicationEntity).
+Response Content Type: JSON - [SourceApplicationEntity](xref:Nvidia.Clara.DicomAdapter.API.SourceApplicationEntity).
 
 | Code | Description                       |
 | ---- | --------------------------------- |
 | 200  | AE Titles retrieved successfully. |
+| 404  | AE Titles not found.              |
 | 500  | Server error.                     |
 
 ### Example Request
 
 ```bash
-curl --location --request GET 'http://localhost:5000/api/config/destinationaetitle'
+curl --location --request GET 'http://localhost:5000/api/config/sourceaetitle/pacs'
 ```
 
 ---
 
-## GET /api/config/sourceaetitle/{ae-title}
+## POST /api/config/sourceaetitle
 
-Retrieves the named source AE Title.
+Creates a new Source AE Title.
 
 ### Parameters
 
-| Name     | Type   | Description                   |
-| -------- | ------ | ----------------------------- |
-| ae-title | string | the AE Title to be retrieved. |
+Please see the [SourceApplicationEntity](xref:Nvidia.Clara.DicomAdapter.API.SourceApplicationEntity)
+class definition for details.
 
 ### Responses
 
 Response Content Type: JSON - [SourceApplicationEntity](xref:Nvidia.Clara.DicomAdapter.API.SourceApplicationEntity).
 
-| Code | Description                       |
-| ---- | --------------------------------- |
-| 200  | AE Titles retrieved successfully. |
-| 404  | AE Titles not found.              |
-| 500  | Server error.                     |
+| Code | Description                    |
+| ---- | ------------------------------ |
+| 201  | AE Title created successfully. |
+| 400  | Validation error.              |
+| 500  | Server error.                  |
 
 ### Example Request
 
 ```bash
-curl --location --request GET 'http://localhost:5000/api/config/sourceaetitle/pacs'
+curl --location --request POST 'http://localhost:5000/api/config/sourceaetitle' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+	"hostIp": "1.2.3.4",
+	"aeTitle": "Orthanc"
+
+}'
+```
+---
+
+## DELETE /api/config/sourceaetitle/{ae-title}
+
+Deletes a Source AE Title.
+
+### Parameters
+
+| Name | Type   | Description                   |
+| ---- | ------ | ----------------------------- |
+| name | string | the AE Title to be retrieved. |
+
+### Responses
+
+Response Content Type: JSON - [SourceApplicationEntity](xref:Nvidia.Clara.DicomAdapter.API.SourceApplicationEntity).
+
+| Code | Description         |
+| ---- | ------------------- |
+| 200  | AE Title deleted.   |
+| 404  | AE Title not found. |
+| 500  | Server error.       |
+
+### Example Request
+
+```bash
+curl --location --request DELETE 'http://localhost:5000/api/config/sourceaetitle/pacs'
 ```
 
 ---
 
-## GET /api/config/destinationaetitle/{name}
+## GET /api/config/destinationaetitle
 
-Retrieves the named destination AE Title.
+Retrieves a list of Destination AE Titles.
```

Comment:
Does it contain AET only or the AE details including AET. See, not being precise about AE or AET could cause confusions.

Reply:
Fixed